tiled/server/authentication.py

```diff
@@ -1095,12 +1116,13 @@ async def whoami(
     )
 
 
-@base_authentication_router.post("/logout")
+@base_authentication_router.post("/logout", include_in_schema=False)
```

Comment:
This might be a moot point because it's already removed for the schema, but wanted to point out here the `deprecated` keyword as a possibility.

```suggestion
@base_authentication_router.post("/logout", include_in_schema=False, deprecated=True)
```

Reply:
Moot but good to know about!